lib/ex_doc/formatter/html/templates.ex

```diff
@@ -120,6 +120,11 @@ defmodule ExDoc.Formatter.HTML.Templates do
     |> h()
   end
 
+  @doc false
```

Comment:
I think this function can be private because the template is executed in this module? Or have you ran into issues?